crates/rolldown/src/module_loader/module_loader.rs

```diff
@@ -76,11 +72,7 @@ impl ModuleLoader {
     fs: OsFileSystem,
     resolver: SharedResolver,
   ) -> anyhow::Result<Self> {
-    // 1024 should be enough for most cases
-    // over 1024 pending tasks are insane
-    let (tx, rx) = tokio::sync::mpsc::channel::<Msg>(1024);
-
-    let tx_to_runtime_module = tx.clone();
+    let tx_to_runtime_module = Arc::clone(&plugin_driver.tx);
```

Comment:
I don't have a solid reason, but this doesn't seems good. Do we have to place the channel in the `PluginDriver`?

Reply:
If there not store channel at the `PluginDriver`, it needs to set `tx` to `Arc<PluginDriver>`, mutate the `Arc<PluginDriver>` is a little messy. Let me try to fixed it.

Reply:
Fixed